pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -57,23 +58,18 @@ func (r *Runner) Prepare(logger kitlog.Logger, config *rest.Config) error {
 
 func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 	Describe("Consoles", func() {
-		var client clientset
-
-		BeforeEach(func() {
-			logger.Log("msg", "starting test")
-
-			client = newClient(config)
-
-			// Wait for MutatingWebhookConfig to be created
-			Eventually(func() bool {
-				_, err := client.Admissionregistration().MutatingWebhookConfigurations().Get("theatre-workloads", metav1.GetOptions{})
-				if err != nil {
-					logger.Log("error", err)
-					return false
-				}
-				return true
-			}).Should(Equal(true))
-		})
+		logger.Log("msg", "starting test")
+
+		client := newClient(config)
```

Comment:
It may also be detrimental to be only calling this once, rather than once per spec, as well.

IIRC the client does some caching, so we might now be seeing pollution between test cases.

Reply:
Cool will revert this just to be safe as it doesn't really matter.